app/routes.py

```diff
@@ -16,7 +17,8 @@ def index():
 
 @app.route("/login")
 def login():
-    return render_template("wip.html")
```

Comment:
Flask has built in login validation that we can use for now to avoid the current error page. See the example below

@app.route("/login", methods=["GET", "POST"])
def login():
    form = LoginForm()
    if form.validate_on_submit():
        flash(
            "Login requested for user {}. remember_me={}".format(
                form.username.data, form.remember_me.data
            )
        )
        return redirect(url_for("index"))

    return render_template("login.html", title="Sign In", form=form)